src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -452,23 +465,38 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
+        /* The instructor preamble aims at informing the instructor about an email that has been sent to the students
+        of his course */
```

Comment:
@damithc I have to disagree with this. That format is, as mentioned, reserved for Javadoc and local variables do not need such.
However, this comment format `/* ... */` is not preferred either. If this comment is to be retained, change it to:
```java
// The instructor ...
// of his course
```